README.md

```diff
@@ -71,8 +91,11 @@ MSALInteractiveTokenParameters *interactiveParams = [[MSALInteractiveTokenParame
 The master branch has been copied over to main branch. The master branch will contain updates only until version 1.2.14, for further releases please refer to 'main' branch instead of 'master'.
 
 ## Installation
+
 ### Using CocoaPods
 
+**For browser-delegated authentication using MSAL iOS and macOS**
```

Comment:
I'd avoid saying "using MSAL iOS and macOS" for this title and "using MSAL iOS" in the native auth title. If we want to say that macOS isn't supported for native auth, we should say it explicitly

Reply:
Got you, I'll update this section as we've explicitly stated this in the beginning paragraphs of the readme